sdk/keyvault/keyvault-certificates/.eslintignore

```diff
@@ -1,2 +1,2 @@
 # Ignoring the core files since they're auto-generated. Eventually, the auto-generated code will be on par with our current eslint rules, but in the mean time we should ignore them.
-src/core
+src/generated
```

Comment:
Do we need the .eslintignore file anymore since folders with the name `generated` are ignored out of the box by our eslint plugin?

Reply:
Yes! you're right. Thank you